userguide/tutorials/aws.adoc

```diff
@@ -20,6 +20,10 @@ Take a look at our http://docs.killbill.io/latest/getting_started.html#_using_ki
 
 == Installation via CloudFormation
 
+The Standalone AMIs are very convenient to start a full stack on one EC2 instance, but for production deployments, we suggest to use our CloudFormation tempate to start a full stack.
```

Comment:
`to start a distributed full stack` maybe, just to distinguish with the other "full stack"?